extensions/amp-story/1.0/amp-story.js

```diff
@@ -733,7 +735,10 @@ export class AmpStory extends AMP.BaseElement {
       attributeFilter: ['class'],
     });
 
-    this.getViewport().onResize(debounce(this.win, () => this.onResize(), 300));
+    this.getViewport().onResize(
+      debounce(this.win, () => this.onResizeDebounced(), 300)
+    );
+    this.getViewport().onResize((size) => this.onResize_(size));
```

Comment:
Nit: Can't we use just one resize observer here (not familiar with how `onResize` works, but still would be neat to have 1 call to `onResize` instead of 2)?

I like that the new resize is not debounced, I wonder if it's needed to debounce the other one here (future action point), or we can optimize what runs on the debounced function to run fast enough we don't need to worry about the performance of that.

Reply:
I'd argue that it's enough of a refactoring for this PR. The `onResizeDebounced()` is called in other places and has implications / things to test that are out of the scope of this PR